docker-compose/config/cse/idcc/cse-idcc-task-manager-application.yml

```diff
@@ -38,11 +38,6 @@ spring:
       definition: consumeMinioEvent;consumeTaskStatusUpdate;consumeTaskEventUpdate
 
 task-server:
```

Comment:
You didn't add the new configuration for spring-cloud-stream, same for d2cc

Reply:
right